pkgs/by-name/bu/bucketloot/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "bucketloot";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "redhuntlabs";
+    repo = "BucketLoot";
+    tag = "v${version}";
+    hash = "sha256-4nH2OpRgFJFi7PqxQgQbUM7gCh8XHWZ6M4LHlVRkywM=";
+  };
+
+  vendorHash = "sha256-4lAi6Rz3d98NT4Omqjbs8xnrmCCzY24oVddUneegN0A=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
```

Comment:
Nothing to change here, but I am asking out of curiosity, that do these flags do? I'm new to the Go side of things